File: tests/fixtures/v2-html-flow/positive/html-flow.js
function direct(req, res) {
  return res.send(req.query.direct)
}

function propagated(req, res) {
  const content = req.query.propagated
  const alias = content
  return res.send(alias)
}

function encoded(req, res, he) {
  const content = he.encode(req.query.encoded)
  return res.send(content)
}

File: tests/fixtures/v2-html-flow/positive/HtmlFlow.cs
class HtmlFlow
{
    void Direct(dynamic Request, dynamic Response)
    {
        Response.WriteAsync(Request.Query["direct"]);
    }

    void Propagated(dynamic Request, dynamic Response)
    {
        var content = Request.Query["propagated"];
        var forwardedValue = content;
        Response.WriteAsync(forwardedValue);
    }

    void Encoded(dynamic Request, dynamic Response)
    {
        var encoded = HtmlEncoder.Default.Encode(Request.Query["encoded"]);
        Response.WriteAsync(encoded);
    }
}
